src/@chakra-ui/gatsby-plugin/semanticTokens.ts

```diff
@@ -38,6 +38,7 @@ const semanticTokens = {
     bodyLight: { _light: "gray.500", _dark: "gray.100" },
     disabled: { _light: "gray.400", _dark: "gray.500" },
     background: { _light: "white", _dark: "gray.700" },
+    quizButton: { _light: "gray.100", _dark: "gray.700" },
```

Comment:
Naming. The name of the color should not be coupled with the implementation name. Should we add this new color to the Design System?

cc @nloureiro